websites/#/라프텔/presence.ts

```diff
@@ -0,0 +1,171 @@
+const presence = new Presence({
+	clientId: "1133347072200949770",
+});
+
+let prevData = "",
+	AnimeData: Detail = {},
+	AnimeDataEpisode: Episode = {};
```

Comment:
Variables like these should use `camelCase` by convention.

Reply:
I did my best